src/lib.rs

```diff
@@ -65,6 +70,13 @@ pub fn run() -> Result<Vec<Warning>, Box<dyn Error>> {
         skip_checks = skip.collect();
     }
 
+    if args.is_present("show-checks") {
+        checks::available_check_names()
+            .iter()
+            .for_each(|name| println!("{}", name));
+        return Ok(vec![]);
```

Comment:
Maybe instead of this we just exit with code 0?

Reply:
This approach seems really more transparent to terminal arguments.  Thank you! I will correct 😊